tools/options.py

```diff
@@ -121,3 +121,12 @@ def extract_profile(parser, options, toolchain, fallback="develop"):
                                 " supported by profile {}").format(toolchain,
                                                                    filename))
     return profile
+
+def not_disabled(parser, mcu):
```

Comment:
Also, is_cortex_a_disabled implies that I'm checking if all cortex-A CPUs are disabled, not this particular CPU.